content/docs/command-reference/commit.md

```diff
@@ -42,10 +42,10 @@ scenarios are further detailed below.
 
 - In cases where we have previously executed a stage (either by writing
   `dvc.yaml` manually and using `dvc repro`, or with `dvc run`), but later
-  notice that some of the output files or directories it creates, which are
-  already in the <abbr>workspace</abbr>, are missing from `dvc.yaml` (`outs`
-  field). We can
-  [add missing outputs to an existing stage](/docs/user-guide/how-to/add-output-to-stage)
+  notice that some of the files/directories used by the stage as dependencies,
+  or created as outputs, which are already in the <abbr>workspace</abbr>, are
```

Comment:
> which are

"which could be" would be more precise but OK, since this whole part is in a `, which (something,` form that is equivalent to parentheses (meaning that it doesn't change the result of the sentence).

Reply:
However these sentences are really long and complicated. Not easy to read...

Reply:
(same in run)

Reply:
To be honest I'm not sure that we need anything at all after the `[add missing outputs to an existing stage]` link. That's the whole point of this item, right? Everything else doesn't add much information unless I'm missing something.

Agree with @jorgeorpinel - it feels that it should be simlpler.

Reply:
Removed these points (https://github.com/iterative/dvc.org/pull/1914#discussion_r526539027):
`either by ..... with dvc run`, `which are already in the workspace`, `deps and outs field respectively`
 Now I think it's much simpler. Also we already have a link to `how-to` which already has all the points I removed,

Reply:
The latest version had some errors. I left a separate comment with a specific suggestion so we can finalize this. Resolving here

Reply:
This is still too complicated to my mind. Very hard to read.

`... , but later  .... user by, which ... are (more explanation here)` - let's try to avoid this.

Let's try to keep sentence structure simple. But in this case you probably don't need even that level of details (like `outs` field?)

Reply:
also an outdated comment, disregard please ^^